src/software/amazon/ion/impl/lite/IonSystemLite.java

```diff
@@ -180,8 +180,7 @@ public SymbolTable getSystemSymbolTable(String ionVersionId)
     public Iterator<IonValue> iterate(InputStream ionData)
```

Comment:
it is not obvious that this method is problematic;  please add a comment indicating why it's deprecated

Reply:
The comment is in the interface javadoc but will add a note here as well.